biom/table.py

```diff
@@ -522,6 +522,39 @@ def matrix_data(self):
         """The sparse matrix object"""
         return self._data
 
+    def to_pandas(self):
+        """
+        Returns
+        ----------
+        counts : pd.SparseDataFrame
+            Pandas representation of counts
+        sample_metadata : pd.DataFrame
+            Pandas representation of the sample metadata
+        observation_metadata : pd.DataFrame
+            Pandas representation of the observation metadata
+
+        Note
+        ----
+        This can only handle
+        """
+        m = self.matrix_data
+        data = [pd.SparseSeries(m[i].toarray().ravel()) for i in np.arange(m.shape[0])]
+        counts = pd.SparseDataFrame(data, index=_bt.ids('observation'),
+                                    columns=_bt.ids('sample'))
+
+        obs_mapping = {}
+        for i in otu_ids:
+            if 'taxonomy' in self.metadata(id=i, axis='observation'):
```

Comment:
That implies that there will need to be a list per column.

It doesn't read very well - but maybe this will be ok for a first pass.
I think this will be readdressed though.